source/components/gitops/APIToken.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
```

Comment:
This file is redundant:
1. We are not using this logic anymore
2. All the logic here is exists in the `auth.tsx` file